.github/workflows/builds.yml

```diff
@@ -190,10 +198,9 @@ jobs:
 
       - name: Setup the build
         run: |
-          . input-leap/macos_environment.sh
```

Comment:
I'm confused. Why has this been removed in the diff? We want to keep CI as identical as can be to the developer build process.

Reply:
This seams to set our Qt path we no longer need to do this.

Reply:
Oh, of course. Yeah. And this PR uses the action now.

Now that's been cleared up, expect we can merge soon.